### PR TITLE
Show current grid also for whoami

### PR DESCRIPTION
**Pull request by hans-d**

### CODE REVIEW

File: cli/lib/kontena/cli/whoami_command.rb [hunk 1]
```diff
@@ -10,10 +10,16 @@ def execute
...
 
     require_api_url
-    puts "Master: #{settings['server']['url']}"
     token = require_token
-    response = client(token).get('user')
-    puts "User: #{response['email']}"
+
+    puts "Master: #{settings['server']['url']}"
+
+    grid = client(require_token).get("grids/#{current_grid}")
```
Comment (jakolehm) [inline cli/lib/kontena/cli/whoami_command.rb]: I think we can skip fetching grid details. `current_grid` already is the grid name?

**Comment by hans-d** [comment]
updated

**Comment by jakolehm** [comment]
LGTM

/cc @nevalla @miskun

**Comment by nevalla** [comment]
Looks good!

**Comment by jakolehm** [comment]
@hans-d this needs rebase.

**Comment by hans-d** [comment]
@jakolehm rebased

**Pull request merged**